### ISSUE TITLE
Question about Group.forward when c == 6

### ISSUE THREAD

**Issue by TRFB** [Pointcept/GPT4Point#37]
Hi, thanks for open-sourcing this great project.

While reading others/point_encoder/point_encoder_with_color.py(line 104), I noticed the following code in Group.forward():

```
if c == 6:
    xyz = xyz[:, :, :3]
    rgb = xyz[:, :, -3:]
```

Since xyz has already been truncated to the first 3 channels, rgb = xyz[:, :, -3:] would actually return the same values as xyz, instead of the original RGB channels.

Later the code concatenates them again:

`xyz = torch.cat((xyz, rgb), dim=-1)`

So I was wondering whether this behavior is intended, or if RGB was supposed to be extracted from the original tensor (e.g., rgb = xyz[:, :, 3:]) before truncating xyz.

Thanks!
